app/src/components/pages/Home.tsx

```diff
@@ -1,7 +1,34 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
 
 function Home() {
-  return <div>Home</div>;
+  let navigate = useNavigate();
+  const routeChange = () => {

```

Comment:
This is a nitpick but I think you can just inline this in the onClick event like `onClick={() => navigate('/login')`. Reduces a bit of clutter and for pages with multiple links it'll be good to keep extraneous hooks down imo.